### ISSUE TITLE
i18n implementation

### ISSUE THREAD

**Issue by bruno12mota** [relax/relax#41]
Best way to implement internationalization opened for discussion

**Comment by mpipet** [comment]
Tried react-intl on a couple of projects, really easy  to implement.

**Comment by bruno12mota** [comment]
That looks pretty cool, thanks for the tip

**Comment by magalhas** [comment]
I'm wondering more on how i18n should be handled UI/UX wise.

**Comment by magano227** [comment]
L10ns seems to be a good option also.
http://l10ns.org/

**Comment by mpipet** [comment]
@magalas  yes its a piece of challenge, how to handle plurals forms or variables in the middle of strings. Technicaly there are plenty of libs to do that.

But ui config remains very difficult to handle.

**Comment by littletower** [comment]
+1

**Comment by sedubois** [comment]
+1 😊
